subprojects/docs/src/docs/stylesheets/userGuidePdf.xsl

```diff
@@ -1,60 +0,0 @@
-<!--
-  ~ Copyright 2010 the original author or authors.
-  ~
-  ~ Licensed under the Apache License, Version 2.0 (the "License");
-  ~ you may not use this file except in compliance with the License.
-  ~ You may obtain a copy of the License at
-  ~
-  ~      http://www.apache.org/licenses/LICENSE-2.0
-  ~
-  ~ Unless required by applicable law or agreed to in writing, software
-  ~ distributed under the License is distributed on an "AS IS" BASIS,
-  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-  ~ See the License for the specific language governing permissions and
-  ~ limitations under the License.
-  -->
-<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="1.0">
-    <xsl:import href="xhtml/docbook.xsl"/>
-    <xsl:import href="userGuideHtmlCommon.xsl"/>
-
-    <xsl:output method="xml"
-                encoding="UTF-8"
-                indent="no"/>
-
-    <!-- Use custom <head> content, to include stylesheets and bookmarks -->
-
-    <xsl:template name="output.html.stylesheets">
-        <link href="base.css" rel="stylesheet" type="text/css"/>
```

Comment:
Since we don't seem to be using this stylesheet any more, how about removing it? :wink: It took me a while to work out why my changes weren't having an effect.

Reply:
We still use it for the DSL reference I think. We'll have to convert that to asciidoctor someday...

Reply:
Ah, you're right. At least I know what to do now.

The DSL Reference probably needs a rethink, not just a conversion, particularly in light of Kotlin DSL.